### DISCUSSION TITLE
Lab: Setting up and running functional tests - Duration, overview and objectives

### DISCUSSION THREAD

**Discussion by tiagocostapt** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#605] [General]
This lab is scheduled for 60 minutes. It should be revied as students don't need that much time.
On the slide we should also review the Lab Overview and Objectives as most of the topics are not covered on the actual current lab.

**Comment by LuizMacedo** [comment]
I'll definitely review this lab next week, as I'm currently working on retiring Azure Passes. As always, thank you for your great contributions @tiagocostapt!
